test/rectangular.jl

```diff
@@ -4,24 +4,38 @@ using Photometry.Aperture: edges,
 @testset "Apertures" begin
 
     ap_rectangle = RectangularAperture(0, 0, 10, 5, 50)
+    @test edges(ap_rectangle) == (-5, 5, -3, 3)
     @test bbox(ap_rectangle) == (-5, 5, -3, 3)
+    @test size(ap_rectangle) == (7, 11)
 
     ap_rectangle_annulus = RectangularAnnulus(0, 0, 15, 20, 19, 21, 50)
+    @test edges(ap_rectangle_annulus) == (-10, 10, -11, 11)    
     @test bbox(ap_rectangle_annulus) == (-10, 10, -11, 11)
-
-    
+    @test size(ap_rectangle_annulus) == (23, 21)
 end
 
 @testset "Rectangular Aperture" begin
 
-    r1 = RectangularAperture(0, 0, 15, 17 , 0)
-    @test sprint(show, r1) == "RectangularAperture(0, 0, w=15, h=17, theta=0)"
+    r0 = RectangularAperture(0, 0, 15, 17 , 0)
+    @test sprint(show, r0) == "RectangularAperture(0, 0, w=15, h=17, theta=0)"
+
+    r1 = RectangularAperture(2, 10, 15, 19 , 50)
+    @test sprint(show, r1) == "RectangularAperture(2, 10, w=15, h=19, theta=50)"
+
+    r = RectangularAperture(0, 0, 5, 10, 10)
+    @test mask(r, method = :center) == mask(r, method = (:subpixel, 1))
```

Comment:
Works fine on my machine but CI gives a bound error. Any suggestions ? @mileslucas

Reply:
Also works fine on my machine. We can investigate this later

Reply:
Okay. Can you review the other changes?